src/assisted_test_infra/test_infra/controllers/node_controllers/tf_controller.py

```diff
@@ -101,6 +101,7 @@ def shutdown_all_nodes(self) -> None:
         for node in nodes:
             self.shutdown_node(node.name)
 
+    @abstractmethod
```

Comment:
Why annotate it with `abstractmethod`? Not sure it's needed here

Reply:
ah ok, I thought it was required since we override it

Reply:
Nope, `abstractmethod` raise an exception in the inheriting class if the abstract method is not implemented. It's basically force the inheriting classes to override this method